### ISSUE TITLE
The more buttons on the home page are disappointing

### ISSUE THREAD

**Issue by allanday** [flathub/linux-store-frontend#115]
The front page shows 6 apps in each category. Clicking the more button for each one shows the following number of apps:

- Popular: 50
- New and updated: 10
- Editors choice: 10
- Editors choice: 10

So in most cases, someone only sees 4 more apps when they click more. This is disappointing and not a great experience.

The obvious fix here would be to show more apps. Since each row contains 7 apps, picking a multiple of 7 would ensure a better page layout. 35 might be a good number (ie. 5 rows).

**Comment by jgarciao** [comment]
I agree the editor's choice are very disappointing. On #69 we discussed fixing the list but I haven't find the time. Could you help me on this?

News and updated show apps that where added or updated in the last 7 days. The only way to make it bigger is making the period longer (15 days?) or telling developers to publish/update more :-)

The most popular display resolutions according to Google Analytics are 1920x1080 (34% users) (7 tiles per row) and 1366x768 (26% users) (5 tiles per row). I think 35 apps per list is a good number
